### PR TITLE
JBPM-7103 - KIE Server custom queries exposed as DataSet editor in wo…

### PR DESCRIPTION
**Pull request by mswiderski**
…rkbench

depends on https://github.com/kiegroup/droolsjbpm-integration/pull/1407

@dgutierr @cristianonicolai could you please have a look?

### CODE REVIEW

**Comment by cristianonicolai** [comment]
jenkins please retest this

File: jbpm-wb-kie-server/jbpm-wb-kie-server-api/src/main/java/org/jbpm/workbench/ks/integration/RemoteDefJSONMarshaller.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+package org.jbpm.workbench.ks.integration;
+
+import static org.dashbuilder.dataset.json.DataSetDefJSONMarshaller.ALL_COLUMNS;
+import static org.dashbuilder.dataset.json.DataSetDefJSONMarshaller.isBlank;
+
+import org.dashbuilder.dataset.json.DataSetDefJSONMarshallerExt;
+import org.dashbuilder.json.JsonObject;
+
+public class RemoteDefJSONMarshaller implements DataSetDefJSONMarshallerExt<RemoteDataSetDef> {
```
[the remark this answers is not in the document]
Comment (mswiderski) [reply jbpm-wb-kie-server/jbpm-wb-kie-server-api/src/main/java/org/jbpm/workbench/ks/integration/RemoteDefJSONMarshaller.java]: will add

**Comment by mswiderski** [comment]
> My only concern is related to using widgets in the views you created. We're trying to get away of that and for anything new, we should always use Errai UI whenever is possible.

I made this to be as similar as the one in dash builder to be consistent so I'd prefer to leave it as is and refactor whenever this will be refactored in dash builder as well. Way easier to follow and understand the logic. And frankly speaking I have no idea how to make it the "new way" :(

**Comment by mswiderski** [comment]
@cristianonicolai changes applied

**Pull request merged**